packages/rln/src/rln_light.ts

```diff
@@ -0,0 +1,235 @@
+import { hmac } from "@noble/hashes/hmac";
+import { sha256 } from "@noble/hashes/sha256";
+import { Logger } from "@waku/utils";
+import { ethers } from "ethers";
+
+import { LINEA_CONTRACT } from "./contract/constants.js";
+import { RLNLightContract } from "./contract/rln_light_contract.js";
+import { IdentityCredential } from "./identity.js";
+import { Keystore } from "./keystore/index.js";
+import type {
+  DecryptedCredentials,
+  EncryptedCredentials
+} from "./keystore/index.js";
+import { KeystoreEntity, Password } from "./keystore/types.js";
+import {
+  buildBigIntFromUint8Array,
+  extractMetaMaskSigner
+} from "./utils/index.js";
+
+const log = new Logger("waku:rln");
```

Comment:
I would use different logger prefix, maybe `waku:credentials` or something 
It's because I believe this file is not RLN per se, so we should improve context here